#VOV-5562

yingfeng-iu opened the pull request.

yingfeng-iu committed:
VOV-5562: add logic to enable security token checking to handle requests
sent from multiple urls

yingfeng-iu committed:
VOV-5562:
- revert files line ending to Unix format;
- change also Application.xml under stage/ dir
- temporarily change debug log to INFO to make sure they show up in logs

yingfeng-iu committed:
VOV-5562:
- add avalon alias for localhost to the URL whitelist, so that wowza is
able to send authorize request to Avalon instead to itself (localhost)
- some enhancement on parsing the URL whitelist to handle whitespace and
slash at end of URL etc, also skip invalid urls instead of abadoning the
rest of the list.

yingfeng-iu committed:
VOV-5562 added instructions for packaging the wowza extension in READ
file.

yingfeng-iu committed:
VOV-5562 correct typos

bkeese reviewed (approved):
Logically, this all looks good to me.
The PR would be cleaner if there weren't so many whitespace changes and if indentation was consistent.

yingfeng-iu committed:
VOV-5562: adding https urls to the whitelist

yingfeng-iu committed:
VOV-5562: correct indentation to be consistent

yingfeng-iu commented:
The indentation inconsistency was probably due to my Eclipse settings use 4 spaces while the original was 2 spaces. I just corrected that to be all the same. It looks wierd in Github somehow, even though it looks fine in Eclipse editor. Use the ignore white space option to view changes might be clearer.

phuongdh commented:
@yingfeng-iu if you change it back to be all 2 spaces it'd be much easier to track which lines have been modified, as of now it looks like everything has been edited.

Also we should squash these commits into 1.

yingfeng-iu committed:
VOV-5562
- changed tab/space indentation to be consistent with original format
- added classpath variable to avoid using absolute path in .classpath
- updated README release version etc

yingfeng-iu committed:
VOV-5562 convert tab to 2 spaces in java file
